timemachine/fe/absolute_hydration.py

```diff
@@ -156,3 +156,125 @@ def log_prob(xs, lam):
     resample = partial(conditional_multinomial_resample, thresh=resample_thresh)
 
     return samples, lambdas, propagate, log_prob, resample
+
+
+def set_up_ahfe_for_smc_parameter_changes(
```

Comment:
Naming: is `ahfe` accurate?

(Perhaps `set_up_smc_parameter_changes_at_endstates` or similar?)

Reply:
updated